src/app/inventory/dimItemService.factory.js

```diff
@@ -588,6 +588,7 @@ export function ItemService(
     }
 
     if (item.owner === store.id) {
+      // TODO: handle space for postmaster
```

Comment:
we'll want to reuse the logic we already have so that we can move an item and have it auto-move other items to make space.